web/beacon-app/src/features/topics/hooks/useFetchTopic.ts

```diff
@@ -0,0 +1,22 @@
+import { useQuery } from '@tanstack/react-query';
+
+import axiosInstance from '@/application/api/ApiService';
+import { RQK } from '@/constants/queryKeys';
+
+import { topicRequest } from '../api/topicDetailApiService';
+import { TopicQuery } from '../types/topicService';
+
+export function useFetchTopics(topicsID: string): TopicQuery {
+  const query = useQuery([RQK.TOPIC, topicsID], () => topicRequest(axiosInstance)(topicsID), {
```

Comment:
If `topicsID` is replaced with `topicID` above, then it should be replaced here as well.